README.adoc

```diff
@@ -161,18 +161,18 @@ You need a running *rabbitmq* in docker. To achieve this run:
 
 You need a running *s3* compatible objectstorage in docker. To achieve this run:
 
-    $ docker run -d --env 'REMOTE_MANAGEMENT_DISABLE=1' --env 'SCALITY_ACCESS_KEY_ID=accessKey1' --env 'SCALITY_SECRET_ACCESS_KEY=secretKey1' --name=s3 zenko/cloudserver:8.2.6
+    $ docker run -d -p 8080:8000 --env 'REMOTE_MANAGEMENT_DISABLE=1' --env 'SCALITY_ACCESS_KEY_ID=accessKey1' --env 'SCALITY_SECRET_ACCESS_KEY=secretKey1' --name=s3 zenko/cloudserver:8.2.6
```

Comment:
Adding back the port binding seems unrelated with the commit title `JAMES-3187 remove separate chapters "Handling attachment indexing"`